xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -111,6 +111,21 @@ void CSelectionStreams::Clear(StreamType type, StreamSource source)
   }
 }
 
+void CSelectionStreams::Clear(StreamType type, int source)
```

Comment:
Because I only want to clear/remove the stream associated with the give source id